hot-reload.js

```diff
@@ -31,7 +39,7 @@ const watchChanges = (dir, lastTimestamp) => {
 
     timestampForFilesInDirectory (dir).then (timestamp => {
 
-        if (!lastTimestamp || (lastTimestamp === timestamp)) {
+        if (!timestamp || !lastTimestamp || (lastTimestamp === timestamp)) {
```

Comment:
In what cases `timestamp` is nonexistent?

Reply:
@xpl Since we generate `timestamp` from the set of files filtered by `includes`, it may end up an empty string (`''` from `join`ing `[]`), which is falsy. In this case, since not all the required files exist, we want to execute the timeout again, to wait for them to exist before reloading.